app/elements/qea-property-analyzer/qea-property-analyzer.html

```diff
@@ -0,0 +1,138 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/hydrolysis/hydrolysis-analyzer.html">
+<link rel="import" href="../../behaviors/qea-inspect-behavior.html">
+
+<!-- use this element as a library to analize a polymer elements -->
+<dom-module id="qea-property-analyzer">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+
+    <hydrolysis-analyzer id='hydro'
+         src=[[src]]
+         analyzer={{_analyzer}}
+         transitive=[[recursively]]
+         loading={{loading}}
+         clean>
+    </hydrolysis-analyzer>
+
+  </template>
+  <script>
+  (function () {
+    'use strict'
+
```

Comment:
I assume all the properties and methods that don't start with an `_` are supposed to be accessible publicly right?